include/phasar/PhasarLLVM/ControlFlow/LLVMBasedCFG.h

```diff
@@ -49,10 +49,11 @@ template <typename Derived> class LLVMBasedCFGImpl : public CFGBase<Derived> {
   }
 
 protected:
-  LLVMBasedCFGImpl(bool IgnoreDbgInstructions = true) noexcept
+  LLVMBasedCFGImpl() noexcept = default;
+  LLVMBasedCFGImpl(bool IgnoreDbgInstructions) noexcept
       : IgnoreDbgInstructions(IgnoreDbgInstructions) {}
 
-  bool IgnoreDbgInstructions = false;
+  bool IgnoreDbgInstructions = true;
```

Comment:
are there more locations that configure this behaviour? Is it the right thing to make this change in this PR?

Reply:
That was already the default, because the ctor parameter `IgnoreDbgInstructions` was set to true by default. THis change just makes it clearer what the actual default really is